_data/releases.yaml

```diff
@@ -193,71 +258,6 @@
       items:
         - format: Notes
           url: https://wildfly.org/news/2021/03/11/WildFly23-Final-Released/
-- version: 23.0.0.Beta1
```

Comment:
Ah deleting that wasn't in the release process ;-)

Reply:
Thanks for adding it.